.cache/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
make sure the directory for the cache is present

Reply:
Why can't this directory be created automatically? I would prefer not to have an empty folder in the project

Reply:
I'll see if I can make that happen